### PR TITLE
added transactional annotation to method

### PR DESCRIPTION
**Pull request by georgebyelas**

### CODE REVIEW

File: molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java [hunk 1]
```diff
@@ -23,7 +23,8 @@ public boolean isFinalState()
 		@Override
 		public EnumSet<JobStatus> getStateTransitions()
 		{
-			return EnumSet.of(SCHEDULED, RUNNING);
+			//FIXME temp
+			return EnumSet.of(SCHEDULED, RUNNING, FAILED, COMPLETED, CANCELLED);
```
Comment (dennishendriksen) [inline molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java]: Is this temporary fix still required after fixing the transactional issue?

File: molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java [hunk 2]
```diff
@@ -41,9 +42,10 @@ public boolean isFinalState()
 		}
 
 		@Override
+		//FIXME temp
 		public EnumSet<JobStatus> getStateTransitions()
 		{
-			return EnumSet.of(RUNNING);
+			return EnumSet.of(RUNNING, COMPLETED, FAILED, CANCELLED);
```
Comment (dennishendriksen) [inline molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java]: Is this temporary fix still required after fixing the transactional issue?

File: molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/job/JobServiceImpl.java [hunk 3]
```diff
@@ -37,7 +39,8 @@ public void updateJobStatus(JobStatusUpdate statusUpdate)
 		}
 
 		job.setStatus(statusUpdate.getStatus());
-		job.setOutputMessage(statusUpdate.getOutputMessage());
+//		job.setOutputMessage(statusUpdate.getOutputMessage());
```
Comment (dennishendriksen) [inline molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/job/JobServiceImpl.java]: Please add a note why you disabled this line.

Comment (georgebyelas) [inline molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/job/JobServiceImpl.java]: Now, the message is too large for the field in database

File: molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java (hunk 1 above)
Comment (georgebyelas) [inline molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java]: status transition from RUNNING to CREATED is not allowed. Allowed transitions [COMPLETED, FAILED, CANCELLED]

File: molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java (hunk 2 above)
Comment (georgebyelas) [inline molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java]: status transition from RUNNING to CREATED is not allowed. Allowed transitions [COMPLETED, FAILED, CANCELLED]

File: pom.xml [hunk 4]
```diff
@@ -290,7 +290,7 @@
                     <artifactId>jetty-maven-plugin</artifactId>
                     <version>8.1.12.v20130726</version>
                     <configuration>
-                        <stopPort>${jetty.stop.http.port}</stopPort>
+                        <stopPort>12345</stopPort>
```
Comment (dennishendriksen) [inline pom.xml]: Please revert change

**Pull request merged**